Tests/DatadogTests/Datadog/RUM/RUMMonitor/Scopes/RUMApplicationScopeTests.swift

```diff
@@ -151,7 +151,7 @@ class RUMApplicationScopeTests: XCTestCase {
             backgroundEventTrackingEnabled: .mockAny()
         )
 
-        let simulatedSessionsCount = 200
+        let simulatedSessionsCount = 400
```

Comment:
Using tests repetition in local:
* with 200 samples - 4 failures in 500 runs
* with 300 samples - 1 failures in 500 runs
* with 400 samples - 0 failures in 500 runs

With 400 samples it runs in `~180ms`.